way_to_home/tests/unittests/user_profile/test_models.py

```diff
@@ -0,0 +1,106 @@
+"""
+UserProfile Models Test
+========================
+This module provides complete testing for all UserProfile models functions.
+"""
+
+from django.test import TestCase
+from custom_user.models import CustomUser
+from user_profile.models import UserProfile
+
+
+class UserProfileTestCase(TestCase):
+    """ TestCase for providing UserProfile model testing """
+
+    def setUp(self):
+        """ Method that provides preparation before testing UserProfile model's features. """
+        custom_user = CustomUser.objects.create(
+            id=111,
+            email='user@mail.com',
+            password='1111',
+            is_active=True
+        )
+        custom_user.save()
+
+        CustomUser.objects.create(  # this object use in 'test_create' function
+            id=222,
+            email='seconduser@mail.com',
+            password='2222',
+            is_active=True
+        ).save()
+
+        self.user_profile = UserProfile.objects.create(
+            id=333,
+            first_name='userName',
+            last_name='userSurname',
+            user_id=custom_user.id,
```

Comment:
I think you should pass a `User` object instead of his `id`.

Reply:
Done!